.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -37,7 +37,7 @@ Please do not remove items from the checklist
 - [ ] An OWNER pushes the tag with
       `git push $VERSION`
   - Triggers prow to build and publish a staging container image
-      `gcr.io/k8s-staging-kueue/kueue:$VERSION`
+      `us-central1-docker.pkg.dev/k8s-staging-images/kueue:$VERSION`
```

Comment:
should it be updated to `us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue`?

Reply:
Right, thanks.